tests/functional/test_cli.py

```diff
@@ -338,10 +376,73 @@ def test_execute_with_auto_monitor(self):
         #   status was periodically pooled and returned 'running' until the final 'succeeded' resumes to download.
         raise NotImplementedError
 
+    # NOTE:
+    #   For all below '<>_auto_resolve_vault' test cases, the local file referenced in the Execute request body
+    #   should be automatically handled by uploading to the Vault and forwarding the relevant X-Auth-Vault header.
```

Comment:
I guess that we cannot validate that the file has really be uploaded and accessed with correct header since all of that occurs under the hood? I mean as far as I know the execute function could have copied the file to the output location 😊.

Reply:
If it didn't work, the test would fail because the execution would be missing an input (either missing completely, http not found or http forbidden if missing/bad header).

Reply:
If we wanted, we could had an explicit check that the endpoint was called with some wrapping/mock magic of some function within the upload, but overkill validation in my opinion.